ouroboros-consensus/test-consensus/Test/Consensus/Ledger/Byron.hs

```diff
@@ -215,7 +217,7 @@ test_golden_ChainState = goldenTestCBOR
     exampleChainState
     "test-consensus/golden/cbor/byron/ChainState"
   where
-    exampleChainState = CS.appendEBB secParam windowSize 6 $
+    exampleChainState = CS.appendEBB secParam windowSize 6 hashBytes $
```

Comment:
We should have another test in the other direction: we should be able to decode the chain state. We can have one of these for each chain state version.

Reply:
I have pushed a commit that checks this.